pkg/timelock/timelock.go

```diff
@@ -73,6 +75,11 @@ func NewTimelockWorker(nodeURL, timelockAddress, callProxyAddress, privateKey st
 		return nil, fmt.Errorf("from block can't be a negative number (minimum value 0): got %d", pollPeriod)
 	}
 
+	useBlockOffset := false
+	if strings.Contains(nodeURL, "avalanche-testnet") || strings.Contains(nodeURL, "base-testnet") {
```

Comment:
Can we divide the PR into two? One of them to address the probes and the other to work the block offset.

The probes part looks ready to me.

Regarding the block offset I'm not comfortable hardwiring it to certain networks. It's a "problem" derived from certain RPC servers which don't allow subscribers to subscribe to more than N past blocks to increase performance. We can work it out a different PR.